client/hotrod-client/src/main/java/org/infinispan/client/hotrod/exceptions/ParallelOperationException.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.client.hotrod.exceptions;
```

Comment:
@gdarmont Please make sure your IDE is properly configured to follow out code style, as described in http://infinispan.org/docs/8.0.x/contributing/contributing.html#_setting_up_your_ide